tests/test_filelock.py

```diff
@@ -385,26 +385,13 @@ def decorated_method() -> None:
     assert not lock.is_locked
 
 
-@pytest.mark.skipif(sys.platform != "win32", reason="Tests behavior of UnixFileLock on Windows systems")
-def test_unix_lock_on_windows(tmp_path: Path) -> None:
-    lock = UnixFileLock(str(tmp_path / "lockfile"))
+def test_wrong_platform(tmp_path: Path) -> None:
+    lock_type = UnixFileLock if sys.platform == "win32" else WindowsFileLock
+    lock = lock_type(str(tmp_path / "lockfile"))
 
     assert not inspect.isabstract(  # noqa: SC200
```

Comment:
a better test here would be to instantiate the class and test that calling the acquire/release raises NotImplementedError 🤔